### DISCUSSION TITLE
remote installation for explorerpatcher doesnt work as expected

### DISCUSSION THREAD

**Discussion by tfgmail** [valinet/ExplorerPatcher#2903] [Q&A]
Hi,

 i am trying to deploy the software remotely to our client machine ,using pdq deploy but after installation the system goes black i am not able to see anything ,

but i can see that the installation is success through pdq deploy replot

when the screen is black i need to do a force shutdown and then restart ,to bring it back to normal

but its not an ideal case to do it for all the client machines manually

Could you please help me with this?

Someone could please help me with this

**Comment by Amrsatrio** [comment]
Hi, please let us know which version of EP and which specific build of Windows you're experiencing this problem with.

**Reply by tfgmail** [reply]
Hi,

EP : [22621.3007.63.4]

am facing this issue with windows 11 and 10

but if i do a manual installation it works fine,issue is only with the remote installation and we are using pdq deploy for it

**Comment by pyrates999** [comment]
Are you trying to install EP before you login?

**Reply by tfgmail** [reply]
no am installing it after login and am trying to do it on the domain joined systems ,i mean for the domain users

**Reply by pyrates999** [reply]
The user has to have local admin privileges in order to install EP.  Is the domain user a member of the local administrators group?

**Reply by tfgmail** [reply]
we are deploying it through pdq deploy ,where we have the admin privilages,actually EP gets installed but the only problem is like the screen goes black after that,then we need to do a force restart to get back the system,after the force restart system works fine and we can see that EP is installed in it

and its not practical to force restart all the client machines manually after the installation

**Reply by pyrates999** [reply]
is OOBE running at the time when you install EP?

**Reply by pyrates999** [reply]
You said you have admin privileges already.  What user are you running EP as then?  Is that user a member of the local administrators group?

**Reply by tfgmail** [reply]
on domain users profile they don't have local administrator's privilege

**Reply by tfgmail** [reply]
NO its not running,normally we are installing it in devices which are already in use

**Reply by pyrates999** [reply]
EP requires the user be a member of the local administrators group in order to run after you've installed it.  Try adding the user to the local administrators group before installing EP.

**Reply by tfgmail** [reply]
i tried it with the account having admin privilages but the same issue happens,its getting installed

but the screen goes black after the installation

issues is only when we do remote installation

**Reply by pyrates999** [reply]
Has the user logged in before to that computer?  Or is this the first time they are logging into the computer that you plan to install EP onto?

**Reply by tfgmail** [reply]
NO ITS NOT THE FIRST TIME

**Reply by pyrates999** [reply]
has the user finished logging in before you install EP?

**Reply by tfgmail** [reply]
ya,i did logged into the test machine with the admin rights and then  started install EP remotely using pdq deploy

**Reply by pyrates999** [reply]
when you do that, try the following:

1. start and keep task manager open
2. install EP remotely using pdq deploy
3. after the screen is black, check if explorer.exe is running in task manager's details tab

**Reply by tfgmail** [reply]
the screen is just black ,cant see the task manager also

**Reply by pyrates999** [reply]
try switching to it using alt + tab or windows key + tab.

**Reply by tfgmail** [reply]
both doesnt work,the screen is still black

**Reply by pyrates999** [reply]
can you show the pdq script you're using to install EP?

**Reply by tfgmail** [reply]
ep_setup.exe /silent this is the command line

**Reply by pyrates999** [reply]
not including /silent seems to be doing the same thing for me.  And it is installing itself and restarting explorer.exe for me.  Try it without /silent.

I would check windows event viewer log for any crashes in case they occurred during install.
